src/core.js

```diff
@@ -217,6 +219,27 @@ export default class Editable {
     return new Cursor($host[0], range)
   }
 
+  createRangyRange () {
```

Comment:
It would be better not to expose the rangy range. Is it still needed by the framework?

Reply:
No, I don't need this anymore. It was just super handy to have this while trying out things in the framework. Any reason why you don't want people to use rangy?